examples/coaxial-cable.jl

```diff
@@ -9,17 +9,39 @@ import PlasmaModelingToolkit.Geometry: Segment
 import PlasmaModelingToolkit.Constants: η_0
 import PlasmaModelingToolkit.BoundaryConditions: SurfaceImpedance, PerfectElectricConductor, PerfectMagneticConductor
 import PlasmaModelingToolkit.Sources: CoaxialPort
+import PlasmaModelingToolkit.SVG: Figure, save, svg
 
 domain = AxisymmetricDomain((0, L_coax), (r_coax, R_coax), Air())
 
-inner  = Segment{0.0, 0.0, L_coax, 0.0}()
```

Comment:
Do I understand it correctly that there was a bug in the definition of `inner` and `output` that was detected with the plot?

Reply:
That's right, _r-coordinate_ changes only in a range `(r_coax, R_coax)` so `0.0` is wrong here. After fixing it I realized that normal vector of this segment was pointed in wrong direction (according to our convention that normal vector of a `Segment` is pointed to the left looking from `(x1, y1)` at `(x2,y2)`).